src/core/public/saved_objects/saved_objects_client.ts

```diff
@@ -31,7 +31,10 @@ import {
 import { SimpleSavedObject } from './simple_saved_object';
 import { HttpFetchOptions, HttpSetup } from '../http';
 
-type SavedObjectsFindOptions = Omit<SavedObjectFindOptionsServer, 'namespace' | 'sortOrder'>;
+type SavedObjectsFindOptions = Omit<
+  SavedObjectFindOptionsServer,
+  'namespace' | 'sortOrder' | 'rawSearchFields'
```

Comment:
I omitted `rawSearchFields` here because we only need to be able to use it on the server side.

Despite this, it's still showing up in the public API docs, I suspect due to some quirk of the docs generation process.